terraform/amazon/templates/puppet_agent_user_data.yaml.template

```diff
@@ -32,6 +33,7 @@ write_files:
       fi ;\
       /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy download --output /opt/wing-$${WING_VERSION}/wing --sha256sums https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt  --signature-armored https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt.asc https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/wing_$${WING_VERSION}_linux_amd64'
     ExecStart=/bin/sh -c 'exec /opt/wing-$${WING_VERSION}/wing agent --manifest-url "s3://${puppet_tar_gz_bucket_path}" --cluster-name "${tarmak_cluster}" --instance-name "$$(curl --silent --retry 5 http://169.254.169.254/latest/meta-data/instance-id || echo "unknown")" --server-url "https://bastion.${tarmak_environment}.${tarmak_dns_root}:9443"'
+{{- end }}
```

Comment:
It would be great if we could use the same `ExecStart=` here and have the binary path defined in a Environment variable.

Reply:
So upload the a checksum of the dev binary to s3 as well?

What is the format for generating checksums that Airworthy needs?

Reply:
No I mean you just run the same ExecStart. (The ExecStartPre should still be templated through `If/Else`. THat means in dev mode you don't call airworthy.

In general airworthy is downloaded in a specific version (with it specific hardcoded hash). Then after that everything airworthy downloads is GPG signed